data/templates/containers/registry.tpml

```diff
@@ -0,0 +1,5 @@
+### Autogenerated by /usr/libexec/vyos/conf_mode/containers.py ###
+
+{% if registry is defined and registry is not none -%}
```

Comment:
There is no need to have -%} as this is now handled generally during template rendering. Please remove.

Reply:
> There is no need to have -%} as this is now handled generally during template rendering. Please remove.

Done